internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,51 @@
 package daysteps
 
 import (
-	...
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/mary-ppv/sprint5/internal/personaldata"
+	"github.com/mary-ppv/sprint5/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
-// создайте структуру DaySteps
-...
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
+
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	data := strings.Split(datastring, ",")
+	if len(data) != 2 {
+		return err
+	}
+
+	ds.Steps, err = strconv.Atoi(data[0])
+	if err != nil {
+		return err
```

Comment:
Лучше все ошибки оборачивать
```suggestion
		return fmt.Errorf("conversion error: %v", err)
```